IntoHistory/HeroDetailViewController.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  HeroDetailViewController.swift
+//  IntoHistory
+//
+//  Created by 김민택 on 2022/08/13.
+//
+
+import UIKit
+
+class HeroDetailViewController: UIViewController {
+    
+    // MARK: - Property
+    
+    let identifier = "HeroDetailViewController"
+    
+    // MARK: - View
+    
+    private let popupShape: UIView = {
+        $0.backgroundColor = .white
+        $0.layer.cornerRadius = 16
+        return $0
+    }(UIView())
+    
+    private let closeButton: UIImageView = {
+        let configuration = UIImage.SymbolConfiguration(pointSize: 20)
+        $0.image = UIImage(systemName: "xmark", withConfiguration: configuration)
+        $0.tintColor = .black
+        return $0
+    }(UIImageView())
+    
+    private let heroImage: UIImageView = {
+        $0.image = UIImage(named: "LockedHero.png")
+        $0.contentMode = .scaleAspectFit
+        return $0
+    }(UIImageView())
+    
+    private let heroName: UILabel = {
+        $0.text = "이익수 준장"
+        $0.font = UIFont.boldSystemFont(ofSize: 24)
+        $0.textAlignment = .center
+        return $0
+    }(UILabel())
+    
+    private let heroDescription: UILabel = {
+        $0.text = "1968년 1.21사태 당시 무장공비 31명이 청와대를 습격하자 이를 추적하는 가운데 펼쳐진 포위작전 중 적의 습격으로 전사하였다."
+        $0.font = UIFont.systemFont(ofSize: 17)
+        $0.textAlignment = .center
+        $0.numberOfLines = 0
+        return $0
+    }(UILabel())
+    
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        attribute()
+        layout()
+    }
+    
+    // MARK: - Method
+    
+    private func attribute() {
+        view.backgroundColor = UIColor(red: 0, green: 0, blue: 0, a: 0.5)
+    }
+    
+    private func layout() {
+        view.addSubview(popupShape)
+        
+        popupShape.addSubview(closeButton)
+        popupShape.addSubview(heroImage)
+        popupShape.addSubview(heroName)
+        popupShape.addSubview(heroDescription)
```

Comment:
p5;

이렇게 foreach와 클로저 이용해서도 addsubview가 가능합니다! 이건 그냥 ... 제가 지난번에 사용했던 방식이라 공유드립니당.

```suggestion
        [closeButton, heroImgae, heroName, heroDescription].forEach { popupShape.addSubview($0) }
```

Reply:
오, 이런 방법이 있군요. 좋은 정보 감사합니다~
해당 부분은 5c93193 에서 반영했습니다. 감사합니다~